lib/driver.js

```diff
@@ -13,7 +14,8 @@ class Driver {
     } else {
       const clientOptions = Object.assign(clientOpts, conf.client);
 
-      this._client = new AWS[service](clientOptions);
+      const constr = get(AWS, service);
```

Comment:
why not take in Constructor as an argument, avoid the get (package has not been updated for 3 years by the way, don't know what it looks like vs. source code in repo)

super(AWS.DynamoDB.DocumentCilent)